notebooks/p1_data_client_example.ipynb

```diff
@@ -13,49 +13,47 @@
     "## Prerequisites\n",
     "\n",
     "* Python version >= 3.7\n",
-    "* jupyter notebook\n",
-    "\n",
-    "## Installation\n"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "pycharm": {
-     "is_executing": false,
-     "name": "#%% md\n"
-    }
-   },
-   "source": [
-    "```\n",
-    "!pip install p1_data_api\n",
-    "```"
+    "* jupyter notebook"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Initialization\n",
-    "\n",
-    "```python\n",
-    "\"TOKEN = '<Put your token here>'\",\n",
-    "```"
+    "## Initialization"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 2,
    "metadata": {
     "pycharm": {
      "is_executing": false,
      "name": "#%%\n"
     }
    },
-   "outputs": [],
+   "outputs": [
```

Comment:
Let's create an issue so we won't forget to update this doc after we publish our lib in PyPi.
The error message looks not well.